### PR TITLE
Milestone 3: Mobile Version Portfolio Contact Form

### PR DESCRIPTION
**Pull request by AndrewYacoub**
# Milestone 3: Mobile Version Portfolio 

## In this milestone we have added contact form

- [ ] Add new section form contact form
- [ ] We have add `input` for contact
- [ ] We have add validation to `input` fields
- [ ] Add a `hover`, `valid` and `invalid` CSS property

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -209,6 +209,57 @@ <h2>
...
+                <h1 >CONTACT US</h1>
+            </div>
+            <div class="contact-desc">
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            </div>
+            
+        </div>
+
+        <form action="#" method="post" class="contact-form">
+            <div class="contact-inbox">
```
Comment (ReshyResh) [inline index.html]: - [ ] Please make sure to add the [formspree service](https://formspree.io/html) to collect user data, as per requirement.

File: css/style.css [hunk 2]
```diff
@@ -302,3 +302,211 @@ body {
...
+
+.contact-inbox > input[type="text"]:invalid {
+  border: 4px dashed red;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: rgb(129, 129, 140);
+}
```
Comment (ReshyResh) [inline css/style.css]: - [ ] Please make sure to follow the correct Figma template. Currently, when the page is loaded, the user is greeted with a form where all the forms have a dotted border. 
- [ ] The `line-indent` property of the text makes it stick to the right side currently, please make sure it is centered, as per template requirement.

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi team, 

### Good job so far!
  - ✔Correct use of HTML form validation.
  - ✔Correct use of Flexbox to display elements.
  - ✔No linter error.


#### There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Please check the comments under the review._

----

Cheers and Happy coding! 👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 2 above)
Comment (bushmusi) [reply css/style.css]: @ReshyResh Noted.
I hope now I fix above question

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @Andrew-Ccoder, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**